platform/drivers/baseband/HR_C5000_MDx.cpp

```diff
@@ -251,11 +275,39 @@ uint8_t HR_Cx000< M >::uSpi_sendRecv(const uint8_t value)
             gpio_clearPin(DMR_MOSI);
         }
 
-        delayUs(1);
-        gpio_clearPin(DMR_CLK);
+        delayUs(2);
+
+        gpio_setPin(DMR_CLK);
+
+        delayUs(3);
+
         incoming = (incoming << 1) | gpio_readPin(DMR_MISO);
+
+        gpio_clearPin(DMR_CLK);
+
         delayUs(1);
     }
 
     return incoming;
 }
+
```

Comment:
Sorry, I didn't understand why you added a receive-only function. Isn't the `sendRecv` good enough?